samples/client/petstore/java/okhttp-gson-dynamicOperations/src/main/java/org/openapitools/client/model/BigCatAllOf.java

```diff
@@ -0,0 +1,260 @@
+/*
```

Comment:
can you please merge the latest master into your branch if you've not yet done so?

this file shouldn't be generated in the latest master.

Reply:
i rebased on top of the master but the existing samples were not deleted when calling `./bin/generate-samples.sh`

I removed some samples that were falsely generated. Please let me know if you find more files which should not be generated.